Build/Directory.Build.props

```diff
@@ -0,0 +1,20 @@
+<Project>
+    <PropertyGroup>
+        <TargetFrameworks>netstandard2.0;net471</TargetFrameworks>
+        <MRTKRoot>..\..</MRTKRoot>
+        <UnityInstallRoot>C:\Program Files\Unity</UnityInstallRoot>
```

Comment:
This is only for us to build the dll, but would be based on the project it becomes a part of. Since we aren't packing the unity dlls, the resolution would still be on the consumer.

As an aside, all of these variables can be overridden as build parameters as well

Reply:
More importantly, that means that UnitySetup could be used to provide override parameters for build. It wouldn't be necessary for you, a consumer of the nupkg, to care how we resolve the binary dependency; you would only need to make sure you do. That's where documentation of Unity Versions Tested will be really important. 

If you wanted to include this as a paradigm for your builds, you could override easily. An example is the version number (which is easily visible because of the nupkg output)

```
S:\src\mrtk_ryzngard\Build
[feature/mrtk_packageManager ≡]
>dotnet pack /p:Version=5.4.3-donotpublish
Microsoft (R) Build Engine version 15.8.166+gd4e8d81a88 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  Restore completed in 38.73 ms for S:\src\mrtk_ryzngard\Build\MixedReality.Toolkit\MixedReality.Toolkit.csproj.
  Restore completed in 38.73 ms for S:\src\mrtk_ryzngard\Build\MixedReality.Toolkit.SDK\MixedReality.Toolkit.SDK.csproj.
  MixedReality.Toolkit -> S:\src\mrtk_ryzngard\Build\__output\net471\MixedReality.Toolkit.dll
S:\src\mrtk_ryzngard\Assets\MixedRealityToolkit\_Core\Inspectors\Utilities\Lines\DataProviders\ParabolicConstrainedLineDataProviderInspector.cs(16,36): warning CS0169: The field 'ParabolicConstrainedLineDataProviderInspector.endPointPosition' is never used [S:\src\mrtk_ryzngard\Build\MixedReality.Toolkit\MixedReality.Toolkit.csproj]
S:\src\mrtk_ryzngard\Assets\MixedRealityToolkit\_Core\Inspectors\Utilities\Lines\DataProviders\SimpleLineDataProviderInspector.cs(14,36): warning CS0169: The field 'SimpleLineDataProviderInspector.endPointPosition' is never used [S:\src\mrtk_ryzngard\Build\MixedReality.Toolkit\MixedReality.Toolkit.csproj]
S:\src\mrtk_ryzngard\Assets\MixedRealityToolkit\_Core\Inspectors\Profiles\MixedRealitySpeechCommandsProfileInspector.cs(27,28): warning CS0169: The field 'MixedRealitySpeechCommandsProfileInspector.screenWidth' is never used [S:\src\mrtk_ryzngard\Build\MixedReality.Toolkit\MixedReality.Toolkit.csproj]
  MixedReality.Toolkit -> S:\src\mrtk_ryzngard\Build\__output\netstandard2.0\MixedReality.Toolkit.dll
  MixedReality.Toolkit.SDK -> S:\src\mrtk_ryzngard\Build\__output\net471\MixedReality.Toolkit.SDK.dll
  MixedReality.Toolkit.SDK -> S:\src\mrtk_ryzngard\Build\__output\netstandard2.0\MixedReality.Toolkit.SDK.dll
  Successfully created package 'S:\src\mrtk_ryzngard\Build\__output\MixedReality.Toolkit.SDK.5.4.3-donotpublish.nupkg'.
  Successfully created package 'S:\src\mrtk_ryzngard\Build\__output\MixedReality.Toolkit.5.4.3-donotpublish.nupkg'.
```

Dotnet uses msbuild, so any msbuild paradigms transfer across.

Reply:
I was thinking from being an MRTK developer perspective. If we can fix up the `build.ps1` script to find and forward the Unity install path to `dotnet pack` that would make this transparent for everyone regardless of where Unity is.